lib/prelude.dx

```diff
@@ -1552,3 +1552,6 @@ def throw (_:Unit) : {Except} a =
 
 def assert (b:Bool) : {Except} Unit =
   if not b then throw ()
+
+-- TODO: remove this when we have modules that exercise the data-caching path
+justSomeDataToTestCaching = [toList [1,2,3], toList [], toList [1,2]]
```

Comment:
Does this have to be in the prelude? Could it be in some other fixture module that is imported by `io-tests.dx`? (Maybe in the tests directory with a modified `--lib-path` for running it? But even in the `lib` directory seems better than being in the prelude itself)

Reply:
Good question. Right now the prelude is the only thing that gets cached so it's the only easy way to test it. I'm overhauling the caching system this week so I'd rather just leave this wart here for now and then remove it when that lands.

Reply:
Sounds good. Minor nit: maybe change the comment to say "remove this when modules other than the prelude can exercise the data-caching path" to make it clear that it's about how the compiler treats modules, not about the contents of the modules?

Reply:
Good call. I hadn't noticed that it could be read that way.